docs/pod_scenarios.md

```diff
@@ -1,14 +1,19 @@
 ### Pod Scenarios
-Kraken consumes [Powerfulseal](https://github.com/powerfulseal/powerfulseal) under the hood to run the pod scenarios.
-These scenarios are in a simple yaml format that you can manipulate to run your specific tests or use the pre-existing scenarios to see how it works.
+
+Krkn recently replaced PowerfulSeal with its own internal pod scenarios using a plugin system. You can run pod scenarios by adding the following config to Krkn:
+
+```yaml
+```
 
 #### Pod Chaos Scenarios
```

Comment:
It might be useful to link the schema here or add a list of input parameters supported for user to understand the capabilities and options they can use? Thoughts?